content/sensu-go/6.5/sensuctl/environment-variables.md

```diff
@@ -12,20 +11,72 @@ menu:
+SENSU_INSECURE_SKIP_TLS_VERIFY    skip TLS certificate verification (Boolean value)
+SENSU_NAMESPACE                   namespace in which to perform actions (default "default")
+SENSU_TIMEOUT                     timeout when communicating with sensu backend (default 15s)
+SENSU_TRUSTED_CA_FILE             TLS CA certificate bundle in PEM format
+{{< /code >}}
+
+For example, to quickly check the entities in the `production` namespace while you are currently in the `default` namespace, run:
+
+{{< code shell >}}
+SENSU_NAMESPACE=production sensuctl entity list
+{{< /code >}}
+
+Single-command environment variables are not persistent &mdash; to continue the example, if you run `sensuctl entity list` again, the response will include entities for the `default` namespace (not `production`).
+
+## Set environment variables with sensuctl configure
+
+You can set the following environment variables for sensuctl configure:
 
 {{< code text >}}
-SENSU_API_URL                    URL of the Sensu backend API in sensuctl
-SENSU_NAMESPACE                  Name of the current namespace in sensuctl
-SENSU_FORMAT                     Set output format in sensuctl (for example, JSON, YAML, etc.)
-SENSU_ACCESS_TOKEN               Current API access token in sensuctl
-SENSU_ACCESS_TOKEN_EXPIRES_AT    Timestamp specifying when the current API access token expires
-SENSU_REFRESH_TOKEN              Refresh token used to obtain a new access token
-SENSU_TRUSTED_CA_FILE            Path to a trusted CA file if set in sensuctl
-SENSU_INSECURE_SKIP_TLS_VERIFY   Boolean value that can be set to skip TLS verification
+SENSU_FORMAT                      preferred output format (default "tabular")
+SENSU_NON_INTERACTIVE             do not administer interactive questionnaire
+SENSU_OIDC                        use an OIDC provider for authentication (Boolean value)
+SENSU_PASSWORD                    password
+SENSU_PORT (used with SENSU_OIDC) port for local HTTP web server used for OAuth 2 callback during OIDC authentication (default 8000)
+SENSU_URL                         the sensu backend url (default "http://localhost:8080")
+SENSU_USERNAME                    username
 {{< /code >}}
 
-These examples demonstrate how to use sensuctl to export and set environment variables and configure your shell:
+To set environment variables with sensuctl configure, define the environment variables in the same command.
+For example:
+
+{{< code shell >}}
+SENSU_OIDC=true SENSU_NON_INTERACTIVE=true SENSU_FORMAT=yaml SENSU_PORT=7999 SENSU_TIMEOUT=49s SENSU_URL=http://192.168.7.217:8080 sensuctl configure
+{{< /code >}}
+
+Setting environment variables in a sensuctl configure command allows you to inject the desired environment variables in an automation script, quickly and transparently configuring sensuctl.
+
+Environment variables set with `sensuctl configure` are persistent.
+
+## Set environment variables with sensuctl env
+
```

Comment:
I think the use cases for single-command and configure methods are clear (or at least have some explanation). But with these two other methods, it seems like we need to add some explanation about when and why to use `sensuctl env`.

Reply:
I personally have used `sensuctl env` to export the environment in my shell to the use them with `curl` or other scripts. I'm not sure what the use cases were when this was written.